src/main/java/edu/stanford/protege/webprotege/linearizationservice/mappers/WhoficEntityLinearizationSpecificationMapper.java

```diff
@@ -35,9 +35,9 @@ private List<LinearizationSpecification> extractDefaultSpecificationsFromSpec(Li
         return linearizationSpecifications.stream()
                 .map(spec ->
                         new LinearizationSpecification(
-                                ThreeStateBoolean.FALSE,
-                                ThreeStateBoolean.FALSE,
-                                ThreeStateBoolean.UNKNOWN,
+                                LinearizationStateCell.FALSE,
+                                LinearizationStateCell.FALSE,
+                                LinearizationStateCell.UNKNOWN,
```

Comment:
I think this should be `LinearizationStateCell.FOLLOW_BASE_LINEARIZATION` for telescopic linearizations (and `UNKNOWN` only for non-telescopic ones).
Or, if `LinearizationStateCell.FOLLOW_BASE_LINEARIZATION` is not something that can be saved, then it should be left set to `LinearizationStateCell.UNKNOWN` for telescopic ones (which, I suppose, will result in `LinearizationStateCell.FOLLOW_BASE_LINEARIZATION`, when presented). The point is that by default the telescopic linearization should not override what is set in their root linearization, and future changes to the root linearization should be reflected automatically in the telescopic linearizations as well, unlike they were overwritten by the user in the meantime.